.clang-format

```diff
@@ -0,0 +1,16 @@
+BasedOnStyle:  Google
+
+# Objc allows 100 instead of the traditional 80.
+ColumnLimit: 100
```

Comment:
IMO, 120 is a better value here to encourage descriptive method and variable names. I think 140 would be even better than that, considering how universally-wide monitors are these days (things derived on / close to 80 column limits should have died a decade ago!)

This would also reduce the "noise" in the PR, e.g. less disruption to the blame log.

Reply:
I think it would make sense to go with 120.

Reply:
Changed to 120.